doc_source/ec2-instance-connect-set-up.md

```diff
@@ -17,9 +17,6 @@ For more information about setting up EC2 Instance Connect, see [Securing your b
 + If you configured the `AuthorizedKeysCommand` and `AuthorizedKeysCommandUser` settings for SSH authentication, the EC2 Instance Connect installation will not update them\. As a result, you cannot use Instance Connect\.
 
 **Prerequisites**
-+ **Verify the general prerequisites for connecting to your instance using SSH\.**
```

Comment:
In one of the setup tasks, you need to connect to your instance. Hence, inclusion of this prerequisite. Could you let me know why you recommend deleting this prerequisite?

Reply:
You are right that this prerequisite includes steps required, specifically sections Get information about your instance and Enable inbound traffic to your instance but "Locate the private key" is not required, and actually what EIC is solving (the need to manage those permanent keys). Maybe instead of removing, we could state "Verify the sections Get information about your instance and Enable inbound traffic to your instance from general prerequisites for connecting to your instance using SSH.

Reply:
As agreed in a separate chat, I'll change the heading for this section to "Prerequisites for installing EC2 Instance Connect" to avoid confusion that these prerequisites are for connecting using EC2 Instance Connect. Thank you so much for your input.